pallets/order-book/src/lib.rs

```diff
@@ -259,7 +281,7 @@ pub mod pallet {
 	///  - key1 -> AssetIn
 	///  - key2 -> AssetOut
 	///
-	/// Stores the minimum `buy_amount` of `asset_in` when buying
+	/// Stores the minimum `amount_out` of `asset_out` when buying
```

Comment:
Just to notify, min_fulfillment amount is denominated in `currency_out`

Reply:
@lemunozm can you take care of writing a digest of the changes of the API for the apps team?

Reply:
Sure! I'll do it when I have a solid implementation of this